benchmarks/src/replace.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <benchmark/benchmark.h>
+
+const char src[] =
+    "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam mollis imperdiet massa, at dapibus elit interdum "

```

Comment:
What's the exact source of this lorem ipsum text?

Reply:
Generated via this one https://www.lipsum.com/

Reply:
Seems low risk to me.